### DISCUSSION TITLE
UVtools automatic post processing

### DISCUSSION THREAD

**Discussion by xPew** [sn4k3/UVtools#392] [Ideas]
Hi @sn4k3 !

I saw this FR at PrusaSlicer: https://github.com/prusa3d/PrusaSlicer/issues/5641
Since they mainly optimize and accept changes that benefit their printers in the first place when it comes to new features (understandably...)and it is pending for a year, so would you be interested in make this feature possible in SuperSlicer? I'm glad SuperSlicer got support for profiles recently, that's why I ask you this.
SuperSlicer is a PrusaSlicer+ with user-requested features and other improvements. It is maintained by merill, who is very kind and open to ideas that could make his port better for the users. I think if you could help him with the post-processing coding/user side he can do the programming in the slicer to enable post-processing option to SLA as well. I mentioned him this, and he said he is open to it, but doesn't have a resin printer, and needs some help.
If yes, here is superslicer discord info: https://github.com/supermerill/SuperSlicer/issues/611

**Comment by sn4k3** [comment]
The main problem is they dont have a post-processor per printer, which can be added easy. After that you have everything you need.
Integrate UVtools into SuperSlicer is another topic and require more work, like after slice prompt to open in UVtools. But a simple post-processor would solve it.
Still it's not a huge benefit, because you want to open the file on UVtools anyway to check stuff, and so it will auto convert on open

**Comment by xPew** [comment]
Maybe than somehow add the option to auto open UVtools after saving?

**Reply by sn4k3** [reply]
That can be a option when on SLA mode, and it's possible to auto locate UVtools on Windows

**Reply by xPew** [reply]
This would be useful! May I suggest this to the dev of SuperSlicer, and would you help if its needed? It’s not urgent, as the priority is 2.4 merge, but this would be a nice addition to the future :)

**Reply by sn4k3** [reply]
Sure, np
